pkg/api/api.go

```diff
@@ -94,14 +98,7 @@ func (api *api) Create(namespace string) (playbook.Inventory, error) {
 
 // Delete deletes the inventory, configs and kubernetes namespace for the given namespace.
 func (api *api) Delete(namespace string) error {
```

Comment:
I suggest you to add a param `wait` to this method.

Then, you could have : 

```
if !wait {
 api.deletePlaybook(namespace)
}

//....

func (api *api) deletePlaybook(namespace string) {
    if inv, _ := api.inventories.Get(namespace); inv.Namespace == namespace {
        api.inventories.Delete(namespace)
        api.configs.Delete(namespace)
        log.Println("[WATCHER] Inventories and configs for namespace " + event.Namespace + " was deleted")
    }
}
```

Then in the `WatchDelete()` func, you could have : 

```
api.namespaces.AddListener("http")
// handle delete of inventories and configs files
for event := range api.namespaces.Events("http") {
	if event.Type == "DELETED" {
           api.deletePlaybook(event.Namespace)
        }
}
```